### PR TITLE
Updating packages, commands for unit test coverage, and update to web…

### LINKED ISSUE #1: Clean up the Repo
Npm packages and Github ready.

### PR DESCRIPTION
**Pull request by EdwinGuzman**
…pack for new babel update.

Fixes #1

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -83,27 +72,27 @@ if (process.env.NODE_ENV !== 'development') {
       new webpack.DefinePlugin({
         loadA11y: process.env.loadA11y || false,
         nodeEnv: JSON.stringify('development'),
+        'process.env': {
+          NODE_ENV: JSON.stringify('production'),
```
Comment (ktp242) [inline webpack.config.js]: I think it has assigned `NODE ENV` on Line 75. And I think the value should be 'development' as line 7 indicates.

Comment (EdwinGuzman) [reply webpack.config.js]: Yeah, it's not needed here. I added it accidentally working on a different PR. It's needed when using webpack/babel to build but we don't use webpack/babel to build right now. But it's part of #4.

**Comment by ktp242** [comment]
Sorry I have to withdraw the approvement. I saw the warning on browser console, when I clicked search button on mobile view.

`warning.js:33 Warning: Unknown prop `onDeactivate` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop`

But only the first time after the page was loaded. The second time I click the warning would not show.

**Comment by EdwinGuzman** [comment]
Alright, updated it. Not sure what happened because I didn't update the react focus trap module but the syntax for it got updated.

**Pull request merged**